### ISSUE TITLE
Preview button is not disabled while switching between radio butttons. 

### ISSUE THREAD

**Issue by pushkalkatara** [fossasia/badgeyay#570]
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

While switching between radio buttons 

- Use Background Template
- Add Background Image
- Choose Background Color
the preview button works with Background Template and Background Image but should be disabled when radio button choose background color is selected as it is previewing only the image.

**Expected behavior:**

Either the preview is of the generated PDF, or preview button should be disabled while choose background color radio button is selected.

**Steps to reproduce:**
Disable preview button when radio button choose background color is selected.

**Comment by pushkalkatara** [comment]
I would like to work on the issue, please assign and suggest what can be improved.

**Comment by rupav** [comment]
so issue is basically, preview shown in `background color` is not the same as generated badge, right?

**Comment by pushkalkatara** [comment]
@rupav Yes, also while switching between ` Use Background Template ` and ` Add Background Image ` issue persists.
